server/src/com/thoughtworks/go/server/websocket/ConsoleLogSender.java

```diff
@@ -92,6 +92,11 @@ public void process(final SocketEndpoint webSocket, JobIdentifier jobIdentifier,
             // loop before we've seen the last content update
             if (isRunningBuild) sendLogs(webSocket, streamer, jobIdentifier);
 
+            //send the remaining logs if any
+            if (detectCompleted(jobIdentifier)) {
+                sendLogs(webSocket, consoleService.getStreamer(start, jobIdentifier), jobIdentifier);
```

Comment:
@ketan -- fixed comments
* Close all file streamers after use
* add appropriate logging for the processed logs